spec/requests/posts_spec.rb

```diff
@@ -5,31 +5,31 @@
     before { get '/users/5/posts' }
 
     it 'returns a successful response' do
-      expect(response).to have_http_status(:success)
+      # expect(response).to have_http_status(:success)
     end
 
     it 'renders the index template' do
-      expect(response).to render_template(:index)
+      # expect(response).to render_template(:index)
     end
 
     it 'the body returns a correct placeholder' do
-      expect(response.body).to include('Here is a list of posts for a given user')
+      # expect(response.body).to include('Here is a list of posts for a given user')
     end
   end
 
   describe 'GET #show' do
```

Comment:
- [ ] I think you just commented out your test cases in both files, please make sure to undo this step and fix the tests until they get you all the green number of tests passing in your terminal, take a screenshot and attach it in the PR for the next reviewer to see. 🙏